JiraReporter/AuthorLoader.cs

```diff
@@ -287,5 +304,33 @@ private void SetRemainingEstimate()
             _currentAuthor.RemainingEstimateSeconds = _currentAuthor.InProgressTasksTimeLeftSeconds + _currentAuthor.OpenTasksTimeLeftSeconds;
             _currentAuthor.RemainingEstimateHours = _currentAuthor.RemainingEstimateSeconds / 3600;
         }
+
+        private void SetOverrideEmail()
+        {
+            var author = _policy.UserOptions.Find(u => _currentAuthor.Username == u.JiraUserKey && u.EmailOverride != null);
+            if (author != null)
+                _currentAuthor.EmailAdress = author.EmailOverride;
+        }
+
+        private void SetProjectLead(List<Author> authors)
+        {
+            var lead = authors.Find(a => a.Username == _policy.GeneratedProperties.ProjectManager);
+            if (lead == null)
+            {
+                var projectManager = GetProjectLead(_policy.GeneratedProperties.ProjectManager);
+                authors.Add(projectManager);
```

Comment:
@DanBadea 
Daca e setat din greseala Ignored...am pus verificarea in orice caz. Daca e nevoie, shimb